WcaOnRails/app/controllers/delegate_reports_controller.rb

```diff
@@ -29,6 +29,7 @@ def update
 
     @delegate_report = @competition.delegate_report
     @delegate_report.current_user = current_user
+    @delegate_report.discussion_url = "https://groups.google.com/forum/#!searchin/wca-delegates/subject$3A$20%22" + @delegate_report.competition.name.gsub(" ","$20") + "%22"
```

Comment:
Actually I've just checked that using normal encoding works. Also when you go to url encoded in the standard way, it immediately changes to follow this google encoding. Thus using a the method you mentioned should very well.